apps/org.videolan.VLC/eos-vlc.desktop

```diff
@@ -0,0 +1,14 @@
+[Desktop Entry]
+Version=1.0
+Name=Get VideoLAN Client
```

Comment:
I am not sure about that naming. It will push out the app name to the right (ellipsize) which is the important part for the user to know. I would remove the "Get".